permissions/UDisksListen.permission

```diff
@@ -0,0 +1,23 @@
+# -*- mode: sh -*-
+
+# x-sailjail-translation-catalog = sailfish-catalog
+# x-sailjail-translation-key-description = permission-la-udisks2_listen
+# x-sailjail-description = Access storage device information
+# x-sailjail-translation-key-long-description = permission-la-udisks2_description
+# x-sailjail-long-description = Permissions to listen signals and property changes on storage devices
```

Comment:
What I said applies also here, and even more so if this is in the public list, although I would probably put this to internal list and make it available only together with RemovableMedia.

Reply:
Better to have this only with `RemovableMedia` permission, will change.

Reply:
6fbe565b136b84ccbc77dc5d833ed6cdea3663fa ok?

Reply:
Almost, please drop those translation-catalog values still so that this won't generate translatable strings. I don't think we should have those for internal permissions unless they are intended to be used separately.

Reply:
I was about to ask some point that what is the functionality of that but it makes complete sense now. Removed.

Reply:
Erm, sorry. That translation-catalog key doesn't affect that what I thought it was when I wrote that comment. The idea behind that key is that we can specify in permission file the translation catalog that middleware should read. However currently the only place that defines any permission files is this repository so in practice it's always the same value. Now I realized that translation generator script doesn't use that value at all (although it could check that the value is "sailjail-permissions" for correctness). So it's probably best to not to define any strings here. Alternatively we could improve the script a little.

Reply:
Dropped with bec9f7dd73014d68e81c6aa6c7f95119fe8fac45

Reply:
After a tonload of confusion and adding of 95a38b66ea3843e83dc300b4331b16a4b0ae5668 to the commits I guess it is now as it should or was intended to be.

Reply:
I think it's great. :+1: